docker-compose.yml

```diff
@@ -1,6 +1,25 @@
 version: "2.4"
 
 services:
+  network:
+    image: network
+    build: network
+    container_name: network
+    hostname: network
+    stdin_open: true
+    tty: true
+    cap_add:
+      - NET_ADMIN
+    expose:
+      - "57832"
+    networks:
+      clientnet:
+        ipv4_address: 193.167.0.2
```

Comment:
I'm curious how the network 193.167.0.0/16 was chosen?

Reply:
I just picked the IP ranges used in the [QUIC Interop Runner](https://github.com/marten-seemann/quic-interop-runner/blob/master/docker-compose.yml#L19)